api-common-java/src/main/java/com/google/api/core/ApiFutures.java

```diff
@@ -45,18 +45,35 @@
 public final class ApiFutures {
   private ApiFutures() {}
 
-  /*
-   * @deprecated Use {@linkplain #addCallback(ApiFuture, ApiFutureCallback, Executor) the
-   * overload that requires an executor}. For identical behavior, pass {@link
-   * com.google.common.util.concurrent.MoreExecutors#directExecutor}, but consider whether
-   * another executor would be safer.
+  /**
+   * Registers separate success and failure callbacks to be run when the {@link ApiFuture}'s
```

Comment:
```suggestion
   * Registers a callback to be run when the {@link ApiFuture}'s
```
It's just one callback being passed in, the original doc could confuse people that there are multiple callbacks passed in.

Reply:
done